### PR TITLE
Adding - Logo template and Past Logo page

### LINKED ISSUE #2240: 📄Migration | Logo | Past Logos page
URL: https://ssw.com.au/ssw/logo/Past.aspx

We need to migrate this logo page to our new website.

### Tasks
- [x] Ask @wicksipedia what the URL should be on the new website - maybe https://www.ssw.com.au/logo/past
- [x] Create the page
- [x] Add Front Door redirection from the old to the new page

**Screenshots**
image
**Figure: Past Logos page**

### PR DESCRIPTION
**Pull request by andrewwaltosssw**
Creating new template for `/logo`
Adding past logo assets to Tina from https://www.ssw.com.au/ssw/logo/Past.aspx .

- Affected routes: /logo/past (new route)

- Fixed #2240 

Slot: https://app-sswwebsite-9eb3-pr-2252.azurewebsites.net/ 

- [x] If adding a new page, I have followed the [📃 New Webpage](https://github.com/SSWConsulting/SSW.Website/issues/new?assignees=&labels=&projects=&template=new_webpage.yml&title=%F0%9F%93%84+%7B%7B+TITLE+%7D%7D+) issue template

- [x] Include done video or screenshots
image
**Figure: Old past logos page (left) Vs New past logos page (right)**. 

image
**Figure: Screenshot of new changes on slot**

### CODE REVIEW

File: content/logo/past.mdx [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+            image: /images/company-logos/DataOnion_preview.png
+          - title: Old SSRSAuditor
+            image: /images/company-logos/SSRSAuditor_preview.png
+          - title: OLD SQLDeploy
+            image: /images/company-logos/SQLDeploy-2010_preview.png
+          - title: Old TimePRO
+            image: /images/company-logos/TimePRO-old_preview.png
+          - title: Old .NET User Group
+            image: /images/company-logos/old-NETUGSYD_logo_preview.png
+          - {}
```
Comment (Harry-Ross) [inline content/logo/past.mdx]:
```suggestion
```

File: .tina/collections/logos.tsx [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+          label: "Text",
+          name: "text",
+        },
+        {
+          type: "string",
+          label: "Link",
+          name: "link",
+        },
+      ],
+    }
```
Comment (Harry-Ross) [inline .tina/collections/logos.tsx]: Can you make this a `rich-text` field instead?

File: content/logo/past.mdx (hunk 1 above)
Comment (awaltos) [reply content/logo/past.mdx]: im using an empty object here to insert a blank spot in the grid. The 3 prev blocks require 2 images, hence an empty block followed by 3 images with no title

**Review by Harry-Ross — CHANGES_REQUESTED** [review]
image

Can you also fix the responsive layout on mobile? (maybe so that its just 1 column)

File: content/logo/past.mdx (hunk 1 above)
Comment (Harry-Ross) [reply content/logo/past.mdx]: On other viewports it looks a bit odd:

image

File: .tina/collections/logos.tsx (hunk 2 above)
Comment (awaltos) [reply .tina/collections/logos.tsx]: Done - changed this to a rich-text field and using markdown for the link

File: content/logo/past.mdx (hunk 1 above)
Comment (awaltos) [reply content/logo/past.mdx]: Fixed this by allowing a second 'related' image to each block removing the need for an empty block and also fixing the weird responsive behaviour.

**Comment by andrewwaltosssw** [comment]
> image
> 
> Can you also fix the responsive layout on mobile? (maybe so that its just 1 column)

Done - Made this shift to 2 cols on tablet and 1 on mobile

**Review by Harry-Ross — APPROVED** [review]
lgtm 🚢

**Pull request merged**